app/Services/Forge/ForgeService.php

```diff
@@ -108,9 +108,11 @@ public function markSiteAsNewlyMade(): void
 
     public function generateDatabaseName(): string
     {
-        return Str::limit(
-            Str::slug($this->setting->branch, '_'),
-            64
-        );
```

Comment:
I'm curious if we can can keep using the Str::limit and Str::slug as I got a Forge error "The username format is invalid." having this change.

Reply:
I'm not sure the best way to do this, honestly, this seemed the fastest, but I did not have time to test this yet.  I've just noticed that the names end up not being in sync, and it caused some issues on my end.  I realize the branch naming I have is partly to blame, but I don't think it's terribly unique or out of the ordinary (prefixed branch names like feature/ or fix/ or hotfix/ or whatever).

Can you tell what error you got? @mehrancodes 

thanks!